packages/shared-influx-pieces/package.json

```
@@ -0,0 +1,71 @@
+{
+	"name": "@skyra/shared-influx-pieces",
+	"version": "1.0.0",
+	"description": "The influx pieces for ArchId Network's HTTP-only bots",
+	"author": "@skyra",
+	"license": "Apache-2.0",
+	"main": "dist/index.js",
+	"module": "dist/index.js",
+	"types": "dist/index.d.ts",
+	"type": "module",
+	"exports": {
+		".": {
+			"types": "./dist/index.d.ts",
+			"import": "./dist/index.js"
+		},
+		"./register": {
+			"types": "./dist/register.d.ts",
+			"import": "./dist/register.js"
+		}
+	},
+	"sideEffects": [
+		"./dist/register.js"
+	],
+	"scripts": {
+		"build": "tsc -b src",
+		"watch": "tsc -b src --watch",
+		"typecheck": "tsc -p tsconfig.eslint.json",
+		"lint": "eslint src --ext ts --fix -c ../../package.json",
+		"prepack": "yarn build",
+		"bump": "cliff-jumper",
+		"check-update": "cliff-jumper --dry-run"
+	},
+	"dependencies": {
+		"@sapphire/timer-manager": "^1.0.0",
+		"@skyra/http-framework": "workspace:^",
+		"@skyra/influx-utilities": "workspace:^",
+		"discord-api-types": "^0.37.43",
```

```suggestion
		"discord-api-types": "^0.37.67",
```

That was an old version wew